src/tools/race/CMakeLists.txt

```diff
@@ -1,14 +1,20 @@
 find_package(Threads)
+find_package(LibRt)
 
 if (APPLE)
 	remove_tool(race "race tool does not compile under MAC OS X")
 else()
+	if (NOT LIBRT_FOUND)
+		remove_tool(race "race tool requires librt support")
```

Comment:
> I am not so sure: librt functionality might be present in some libc. Maybe thats even the case in OpenBSD.

How do we find out if that is the case? Do we use some `try_compile` magic again?

> No need to have assumptions about if something links, thats really easy to check.

The assumption was not about whether it links or not but rather if it works without librt.